web-console/4.0-designs/binding/binding.md

```diff
@@ -0,0 +1,30 @@
+# Binding
+
+## Create binding
+![](img/binding-prompt.png)
+- New service instances will have a section at the top that describe what service bindings are for and how to use them. The box will also contain a button to create a service binding for the service instance.
+- This box should disappear once it has been dismissed or a binding has been created.
+![](img/create-binding.png)
+- Clicking on the create button enables the user to select a name for their binding and approve its creation.
+
+## Add secret to application
+![](img/binding-pending.png)
+- Once the binding has been created, the user will be taken to the service bindings tab of the service instance.
```

Comment:
Is it better to just go to the binding page for the new binding?

Reply:
Good call. I'll update